packages/core/contracts/zodiac/OptimisticGovernor.sol

```diff
@@ -218,8 +227,7 @@ contract OptimisticGovernor is Module, Lockable {
     function executeProposal(
         uint256 _proposalId,
         Transaction[] memory _transactions,
-        uint32 _originalTime,
-        SkinnyOptimisticOracleInterface.Request memory _request
+        uint32 _originalTime
     ) public payable nonReentrant {
         // Recreate the proposal hash from the inputs and check that it matches the stored proposal hash.
         uint256 id = _proposalId;
```

Comment:
Minor note on something you didn't change:

Instead of:
```
bytes memory ancillaryData = bytes.concat(bytes("id:"), AncillaryData.toUtf8BytesUint(id));
```

You should be able to use the built-in function to do exactly this:
```
bytes memory ancillaryData = AncillaryData.appendKeyValueUint("", "id", id);
```

I think what you had probably worked as intended, but this is a little more standard across our repo.